portfolio/src/main/webapp/script.js

```diff
@@ -45,8 +45,14 @@ function toggleVisibility (sectionToToggle) {
   }
 }
 
-async function fetchGreeting() {
+async function fetchComments() {
   const response = await fetch('/data');
-  const greeting = await response.text();
-  document.getElementById('fetched-greeting').innerHTML = greeting;
+  const comments = await response.json();
+  const commentsSection = document.getElementById('comments');
+  commentsSection.innerHTML = '';
+  for (var i = 0; i < comments.length; i++) {
+    commentsSection.innerHTML += "<p>";
```

Comment:
Should these be in single quotes? Same for the one below.

https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes